src/commands/appdistribution-distribute.ts

```diff
@@ -79,103 +76,103 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
     const testers = getTestersOrGroups(options.testers, options.testersFile);
     const groups = getTestersOrGroups(options.groups, options.groupsFile);
     const requests = new AppDistributionClient(appId);
-    let app: AppDistributionApp;
-
-    try {
-      const appView =
-        distribution.distributionFileType() === DistributionFileType.AAB
-          ? AppView.FULL
-          : AppView.BASIC;
-      app = await requests.getApp(appView);
```

Comment:
Since we're no longer calling getApp, we aren't able to tell them if they need to onboard their app, or add a contact email. If they need to do either of those things, what's the user experience? Will the metadata extraction fail with a helpful message?

Reply:
Fixed: Checking for 404 now and displaying the old message.